src/timer.rs

```diff
@@ -239,6 +239,10 @@ mod implementation {
 
 #[cfg(feature = "use-mock-crust")]
 mod implementation {
+    // This lint requires to make changes that might introduce discrepancies with the actual Crust code,
+    // so we turn it off for mock crust specifically.
+    #![cfg_attr(feature = "cargo-clippy", allow(needless_pass_by_value))]
```

Comment:
will it be better to put this into lib.rs,  just as what the current master branch do?